index.html

```diff
@@ -1584,17 +1599,10 @@ <h4 id="signed-raw">Signed Raw Transaction</h4>
       const fReDelegateChange = nChange > 1.01 * COIN;
       if (fReDelegateChange) {
         // Enough change to resume cold staking, so we'll re-delegate change to the cold staking address
-        // Ensure the user has an address set - if not, request one!
-        if (!askForCSAddr()) return;
```

Comment:
I had this under `fReDelegateChange` since: when MPW undelegates, it re-delegates change for the user automatically (needing a Cold Address), but if there is **no** change, then everything is sent to a standard pubkey, we don't need a Cold Address.


I think it's redundant to ask the user for a Cold Address if we already know their TX will not delegate anything, right?

Reply:
we can move it back but when the cold staking address is inserted can we just make the transaction go on instead of returning?

Reply:
Sounds good, I don't remember why I made it return with an alert after setting... possible drunk coding night.

Reply:
should be fixed now